pelias/README.md

```diff
@@ -0,0 +1,168 @@
+# Setup guide for Pelias geocoding service.
+
+This guide explains how we set up the geocoding service for https://best.osoc.be.
+
+This folder contains configuration files to download/prepare/build a complete Pelias installation for Belgium
+using the BOSA open address database and various other open data sources.
+It also describes how to set up an apache webserver as a proxy and to make the service serve over [https with certbot](https://certbot.eff.org/).
+This guide was adapted from https://github.com/pelias/docker/tree/master/projects/belgium .
+
+# Why Pelias?
+
+We chose pelias for a couple of reasons:
+* It is open source.
+* It offers a very useful API for forward and reverse geocoding, and even for autocompletion.
+* It provides easy [openaddresses.io](https:openaddresses.io) integration. Which makes it particulary interesting for our project.
```

Comment:
Correct link is `https://openaddresses.io`

Reply:
Fixed as well!